### DISCUSSION TITLE
Is there any way to create beacon light(beam)?

### DISCUSSION THREAD

**Discussion by skies-unbound** [Minestom/Minestom#1410] [Help]
Currently, I am working on a player navigation system. Is there any solution to create a fake beacon beam without placing blocks of the beacon?

**Comment by mworzala** [comment]
You can do it without placing the ore blocks below it by setting the nbt data on the beacon, but you cannot create the light without a beacon block.

**Reply by dev-hydrogen** [reply]
Are we talking about the beam here or the light that the beacon emits here? I was under the impression the beam is client-sided. The minecraft wiki says " Levels: The number of levels available from the pyramid. Immediately changes to the correct value if modified using commands."
Is that just what the vanilla server does?
